packages/ui/components/popover/MeetingTimeInTimezones.tsx

```diff
@@ -34,8 +35,12 @@ const MeetingTimeInTimezones = ({
   endTime,
 }: MeetingTimeInTimezonesProps) => {
   if (!userTimezone || !attendees.length) return null;
-
-  const attendeeTimezones = attendees.map((attendee) => attendee.timeZone);
+  
+  // If attendeeTimezone is unsupported, we fallback to host timezone. Unsupported Attendee timezone can be used due to bad API booking request in the past | backward-compatibility
+  
+  const attendeeTimezones = attendees.map((attendee) => {
+    return isSupportedTimeZone(attendee.timeZone) ? attendee.timeZone : userTimezone;
+  });
```

Comment:
When an unsupported timezone is picked for attendee, we use userTimezone (which is host's timezone) instead.